### PR TITLE
Added time field support

### LINKED ISSUE #187: Add Time Fields support

### PR DESCRIPTION
**Pull request by ezeBalsamo**
Added time field support. In order to do this, I created a DateFieldWebView class, sub classifying FieldWebView. Also, its corresponding Codec (ISOTimeCodec). Now, WebComponentCommandBuilder understands a new message: beTimeInput, collaborating with InputModeCommand for setting the time type.

In addition, I formatted InputModeCommand as* methods, so that they are alligned with community code formatting.
Closes #187

### CODE REVIEW

**Review by gcotelli — DISMISSED** [review]

File: source/Willow-Core-Tests/ISODateCodecTest.class.st
Comment (mtabacman) [inline source/Willow-Core-Tests/ISODateCodecTest.class.st:39]: `#should:raise: ` must not be used unless there is no other alternative, since it can confuse different errors.
All references encountered during development and the new assertions should be changed to:
`#should:raise:withMessageText:`

File: source/Willow-Core-Tests/ISOTimeCodecTest.class.st
Comment (mtabacman) [inline source/Willow-Core-Tests/ISOTimeCodecTest.class.st:46]: See my previous comment about avoiding `#should:raise: `

**Review by mtabacman — DISMISSED** [review]
After the comments are considered, merging of this issue should wait until after #143 is merged into the release-candidate branch, so that any conflicts can also be considered.

**Review by gcotelli — APPROVED** [review]

**Pull request merged**
